packages/components/src/pill/style.scss

```diff
@@ -0,0 +1,8 @@
+.woocommerce-pill {
+	border: 1px solid $medium-gray-text;
```

Comment:
FYI `medium-gray-text` [is getting removed ](https://github.com/WordPress/gutenberg/pull/23648) in the next update of base-styles in https://github.com/woocommerce/woocommerce-admin/pull/4759.

Reply:
Good catch, let's make that `gray-700` if we can :)